packages/contracts-bedrock/package.json

```diff
@@ -29,9 +29,9 @@
     "gas-snapshot": "pnpm build:go-ffi && pnpm gas-snapshot:no-build",
     "snapshots": "npx tsx scripts/generate-snapshots.ts && ./test/kontrol/scripts/make-summary-deployment.sh",
     "snapshots:check": "./scripts/check-snapshots.sh",
-    "slither": "./scripts/slither.sh",
-    "slither:check": "pnpm slither && git diff --exit-code",
-    "slither:triage": "TRIAGE_MODE=1 ./scripts/slither.sh",
+    "slither": "slither .",
+    "slither:triage": "slither . --triage-mode",
+    "slither:fmt": "jq '.' slither.db.json > tmp.json && mv tmp.json slither.db.json",
```

Comment:
The following 1 liner can replace the tempfile approach

```
cat <<< $(jq 'QUERY' sample.json) > sample.json
```

Reply:
Hmm, seems to not outputting correctly and just places the full contents on 1 line. Can debug it more tomorrow

Reply:
Decided we can start by trying github code scanning integration approach to minimize DB management and simplify the triage process